### PR TITLE
Add mappings for item classes

### PR DESCRIPTION
**Pull request by enbrain**

### CODE REVIEW

File: mappings/net/minecraft/item/BundleItem.mapping [hunk 1]
```diff
@@ -12,5 +16,12 @@ CLASS net/minecraft/class_5537 net/minecraft/item/BundleItem
...
 	METHOD method_31563 getBundleOccupancy (Lnet/minecraft/class_1799;)I
 		ARG 0 stack
+	METHOD method_31564 (Lnet/minecraft/class_1799;)I
+		ARG 0 heldStack
 	METHOD method_32343 (Lnet/minecraft/class_1661;Lnet/minecraft/class_1799;)V
 		ARG 1 stack
+	METHOD method_32344 findStack (Lnet/minecraft/class_1799;Lnet/minecraft/class_2499;)Ljava/util/Optional;
+		ARG 0 stack
+		ARG 1 listTag
+	METHOD method_32345 getHeldStacks (Lnet/minecraft/class_1799;)Ljava/util/stream/Stream;
```
Comment (haykam821) [inline mappings/net/minecraft/item/BundleItem.mapping]: Does this returns a stream of items in the `Items` tag? If so, I'd recommend something like `streamContents`, similar to [`Brain#streamMemories`](https://github.com/FabricMC/yarn/blob/fc87d2d7de36b39030ab4e5b1b7ce52e424f37d9/mappings/net/minecraft/entity/ai/brain/Brain.mapping#L119):

```suggestion
	METHOD method_32345 streamContents (Lnet/minecraft/class_1799;)Ljava/util/stream/Stream;
```

File: mappings/net/minecraft/item/BundleItem.mapping [hunk 2]
```diff
@@ -12,5 +16,12 @@ CLASS net/minecraft/class_5537 net/minecraft/item/BundleItem
 		ARG 0 stack
 	METHOD method_31563 getBundleOccupancy (Lnet/minecraft/class_1799;)I
 		ARG 0 stack
+	METHOD method_31564 (Lnet/minecraft/class_1799;)I
+		ARG 0 heldStack
```
Comment (haykam821) [inline mappings/net/minecraft/item/BundleItem.mapping]: What is `heldStack`?

**Review by haykam821 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/BundleItem.mapping (hunk 2 above)
Comment (enbrain) [reply mappings/net/minecraft/item/BundleItem.mapping]: It's an item stack that is held in a bundle. Maybe `contentStack` is better?

**Review by YanisBft — APPROVED** [review]

**Comment by enbrain** [comment]
Closing as most are outdated or superseded.

**Pull request closed without merging**